packages/fiori/cypress/specs/UploadCollection.cy.tsx

```diff
@@ -442,6 +500,44 @@ describe("Events", () => {
 		cy.wrap(terminateEventStub)
 			.should("have.been.called");
 	});
+
+	it("Tests _ondragenter early return when not dragging files", () => {
+		cy.mount(<UploadCollection id="uploadCollection" />);
+
+		cy.get("#uploadCollection")
+			.then(($el) => {
+				const uploadCollection = $el[0] as UploadCollection;
+				
+				// Set initial mode to verify it doesn't change
+				uploadCollection._dndOverlayMode = "None";
+				
+				const emptyDragEvent = {
+					dataTransfer: {
+						items: [],
+						types: [],
+						effectAllowed: 'none'
+					}
+				} as unknown as DragEvent;
```

Comment:
Can you use `new DragEvent(...)` and avoid casting to `unknown`?
Apply this to all places